src/components/test-slot/additional-candidate-details/__tests__/additional-candidate-details.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { TestBed, async, ComponentFixture } from '@angular/core/testing';
+import { IonicModule } from 'ionic-angular';
+import { AdditionalCandidateDetailsComponent } from '../additional-candidate-details';
+import { configureTestSuite } from 'ng-bullet';
+
+xdescribe('VehicleDetailsComponent', () => {
+  let fixture: ComponentFixture<AdditionalCandidateDetailsComponent>;
+  let component: AdditionalCandidateDetailsComponent;
+
+  configureTestSuite(() => {
+    TestBed.configureTestingModule({
+      declarations: [
+        AdditionalCandidateDetailsComponent,
+      ],
+      imports: [IonicModule],
+    });
+  });
+
+  beforeEach(async(() => {
+    fixture = TestBed.createComponent(AdditionalCandidateDetailsComponent);
+    component = fixture.componentInstance;
+  }));
+
+  describe('Class', () => {
+    it('should create', () => {
+      expect(component).toBeDefined();
+    });
+  });
+
+  describe('DOM', () => {});
```

Comment:
empty dom's can be deleted

Reply:
Will need deleting in some pre-existing components as well.